test/integration/arbitrum-mainnet/PriceManipulationTest.t.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.22;
+
+import "forge-std/Test.sol";
+import "./DeploymentLoader.sol";
+import { PriceManipulationLib } from "../utils/PriceManipulation.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract PriceManipulationTest is Test, DeploymentLoader {
```

Comment:
"Manipulation" is typically malicious, since here it's just price changes it should be something like "Movements"